### DISCUSSION TITLE
why `One.A` is not valid?

### DISCUSSION THREAD

**Discussion by abiriadev** [microsoft/typespec#3194] [Q&A] [answered]
I am fairly new to TypeSpec, so currently reading [the language basics guide](https://typespec.io/docs/language-basics/overview).

The what I can't 100% understand is, ['Using namespaces' from Namespaces page](https://typespec.io/docs/language-basics/namespaces#using-namespaces).

> The bindings introduced by a `using` statement are local to the namespace in which they are declared. They do not become part of the namespace themselves.

```ts
namespace One {
  model A {}
}

namespace Two {
  using One;
  alias B = A; // This is valid
}

alias C = One.A; // This is not valid
alias C = Two.B; // This is valid
```

I think `One.A` should also be valid. Can you explain why this is not valid?

Also, I tested the example above locally, and it compiled without any issue.

So, in my think, the proper doc supposed to be

```ts
alias C = Two.A; // This is not valid
alias C = Two.B; // This is valid
```

Am I right? Did I miss something?

**Comment by timotheeguerin** [comment, marked as answer]
yes you are totally right the doc is incorrect and should be that
```
alias C = Two.A; // This is not valid
alias C = Two.B; // This is valid
```
